code/modules/surgery/organs/subtypes/drask.dm

```diff
@@ -0,0 +1,65 @@
+//DRASK ORGAN
+/obj/item/organ/internal/drask
+	name = "drask organ"
+	icon = 'icons/obj/surgery_drask.dmi'
+	icon_state = "innards"
+	desc = "A greenish, slightly translucent organ. /red It is extremely cold."
```

Comment:
That's an oversight right there, I'll remove the /red entirely and replace it with nothing